src/components/DependencyManagementPane/DependencyManagementPane.js

```diff
@@ -140,14 +141,33 @@ class DependencyManagementPane extends PureComponent<Props, State> {
                 </DependencyButton>
               ))}
             </Dependencies>
-            <AddDependencyButton onClick={this.openAddNewDependencyModal}>
-              <IconBase icon={plus} size={20} />
-              <Spacer size={6} />
-              Add New
-              <OnlyOn size="mdMin" style={{ paddingLeft: 3 }}>
-                Dependency
-              </OnlyOn>
-            </AddDependencyButton>
+            <MountAfter
+              delay={1000}
```

Comment:
I chose 1000ms even though a much smaller number would work because I don't want to make the fade-in animation jerky by mutating the DOM during that process. The button is below the fold anyway on mount, so I think it's fine to wait a while.